### PR TITLE
Add Postman tests to pipeline (#189)

### PR DESCRIPTION
**Pull request by matir8**
## Description

- Renamed the action name to `run-tests`
- Added a new job to run the Postman tests in parallel to the existing job for running the unit tests when opening a new PR against `develop`

## Type of change

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I did not update API Controllers, if I did, I added/changed Postman tests
-   [x] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] New and existing unit tests pass locally with my changes
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

1. Open a new PR against `develop`
2. You should see the new GitHub action running automatically

## Link to issue

Closes: #189

### CODE REVIEW

File: API/Dockerfile.dev [hunk 1]
```diff
@@ -10,6 +10,7 @@ WORKDIR /app
 
 COPY . ./
 
+RUN dotnet dev-certs https --clean
```
Comment (Brend-Smits) [inline API/Dockerfile.dev:13]: Why add this?

File: IdentityServer/Dockerfile.dev [hunk 2]
```diff
@@ -12,6 +12,7 @@ WORKDIR /app
 
 COPY . ./
 
+RUN dotnet dev-certs https --clean
```
Comment (Brend-Smits) [inline IdentityServer/Dockerfile.dev:15]: Why add this?

File: .github/workflows/run-tests.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+name: Run tests
```
Comment (Brend-Smits) [inline .github/workflows/run-tests.yml:1]:
```suggestion
name: Run Backend Unit & Integration Tests
```

**Review by Brend-Smits — CHANGES_REQUESTED** [review]
Awesome to see that you got this working. Nice work :)

File: API/Dockerfile.dev (hunk 1 above)
Comment (matir8) [reply API/Dockerfile.dev:13]: To verify that we use newly generated certificates every time we build the containers.

File: IdentityServer/Dockerfile.dev (hunk 2 above)
Comment (matir8) [reply IdentityServer/Dockerfile.dev:15]: To verify that we use newly generated certificates every time we build the containers.

Comment (Brend-Smits) [reply IdentityServer/Dockerfile.dev:15]: Makes sense, thanks!

File: API/Dockerfile.dev (hunk 1 above)
Comment (Brend-Smits) [reply API/Dockerfile.dev:13]: Makes sense, thanks!

File: .github/workflows/run-tests.yml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+name: Run Backend Unit & Integration Tests
+
+on:
+  pull_request:
+    branches: [ develop ]
```
Comment (Brend-Smits) [inline .github/workflows/run-tests.yml:5]:
```suggestion
    branches: [ develop, master ]
```
Actually I forgot about this, just to be sure we should also add this action to PR's to Master (for release)

**Review by Brend-Smits — DISMISSED** [review]

**Review by Brend-Smits — APPROVED** [review]

**Review by niraymak — APPROVED** [review]
Nicely done!

**Pull request merged**